internal/operators/cnv/config.go

```diff
@@ -0,0 +1,27 @@
+package cnv
+
+import (
+	"strings"
+)
+
+type SupportedGPUsDecoder map[string]bool
```

Comment:
Wouldn't a string array be enough? In case it is in there - it is supported.

Reply:
I made a decision to use a map-based set to avoid O(n) linear search in an array in favor of O(1) in a map.